duckduckgo/privacy-dashboard
# support new screen mode 'promptBreakageForm'

https://app.asana.com/0/0/1207195637811331/f



## Description:

Added a new screen mode, `promptBreakageForm`. When the dashboard loads in this mode it will remove the toggle from the breakage form.

## Steps to test this PR:

- `npm run preview`
- select the last option from the dropdown `promptBreakageForm`
- The breakage form should load, but the toggle should be absent - it should look like this:

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/Youbs18YwblmJpLrzsDC/4f0d063a-0ca0-4432-80fc-2e92e0ef6305.png)
